### ISSUE TITLE
Sibling Inserter: the block library doesn't always appear when the "Add block" icon is clicked

### ISSUE THREAD

**Issue by designsimply** [WordPress/gutenberg#11340]
I tested with WordPress 4.9.8 and Gutenberg 4.2.0-rc.1 using Firefox 63.0 on macOS 10.13.6 and found that the sibling inserter ("Add block" icon) doesn't always trigger the block library to appear. The problem _seems_ to happen intermittently and I am not sure yet how to trigger the problem every time.

Steps to reproduce:

1. Create a new post and add a few blocks.
1. At various points during editing, click on the "Add block" icon above each block and check to see if the block library appears.

<strong>Result:</strong> I expected the block library to appear every time I click the "Add block" icon and it doesn't appear every time. ([1m23s](https://cloudup.com/cbpRcXUct5k))

![block-library-does-not-always-appear-when-the-sibling-inserter-is-clicked](https://user-images.githubusercontent.com/1119271/47826562-3adbe300-dd3e-11e8-9e49-6cf6690755b0.gif)
<sup>Seen at http://alittletestblog.com/wp-admin/post.php?post=15423&action=edit&revision=15425 running WordPress 4.9.8 and Gutenberg 4.2.0-rc.1 using Firefox 63.0 on macOS 10.13.6.</sup>

Note to testers: would love an extra check on this one! If you can figure out how to improve the steps for testing for this case, I'd appreciate the help since I haven't been able to trigger the problem reliably so far in my own testing.

**Comment by rossanafmenezes** [comment]
Tested: FireFox, Chrome and Safari on simple site and AT:

I got the same issue only on Safari. Firefox and Chrome work just fine. The block library shows 100% of the time. Using Firefox 63 on macOS (Mojave) and Chrome 70 on macOS (Mojave)
Safari on the other hand (Safari 12 on macOS (Mojave)) doesn't seem to work and shows the same problem as above, showing the block library completely randomly and very, very rarely.
